core/includes/common.inc

```diff
@@ -3312,6 +3312,32 @@ function backdrop_add_html_head_link($attributes, $header = FALSE) {
   backdrop_add_html_head($element, 'backdrop_add_html_head_link:' . $attributes['rel'] . ':' . $href);
 }
 
+/**
+ * Constructs an array of the defaults that are used for styles items.
+ *
+ * @param $data
+ *   (optional) The default data parameter for the style item array.
+ *
+ * @see backdrop_get_css()
+ * @see backdrop_add_css()
+ *
+ * @return array
+ *   An array of key and values.
+*/
+function backdrop_css_defaults($data = NULL) {
+  return array(
+    'type' => 'file',
+    'group' => CSS_DEFAULT,
+    'weight' => 0,
+    'every_page' => FALSE,
```

Comment:
This reminds me of another (mostly unrelated) issue about "every_page_weight": https://github.com/backdrop/backdrop-issues/issues/4942 - they might conflict.

Reply:
Yeah, I saw that. It doesn't look like it'll conflict but we'll see once we try to merge them both.